packages/obojobo-document-engine/documents/empty.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ObojoboDraftDoc>
 	<!-- Sample Obojobo Draft Document -->
-	<!-- https://github.com/ucfcdl/Obojobo-Next/wiki/Obojobo-Document-XML-Format -->
+	<!-- https://ucfopen.github.io/Obojobo-Docs/releases/v3.3.2/authors/ -->
```

Comment:
I could see updating this being a problem in the future so I made https://github.com/ucfopen/Obojobo/wiki/Release-checklist to try to have us remember to update this.